wp-content/themes/core/assets/js/src/utils/dom/smooth-anchors.js

```diff
@@ -19,6 +19,9 @@ const handleAnchorClick = ( e ) => {
 		offset: -150,
 		duration: 300,
 		$target: $( target ),
+		afterScroll: function() {
+			target.focus();
```

Comment:
This is great, except if the element being scrolled to isn't focusable. (such as adding an id attribute to an element which isn't a link or input, etc)  I [think this is a reasonable solution](https://github.com/moderntribe/codet-vision/blob/main/wp-content/themes/core/assets/js/src/utils/dom/smooth-anchors.js#L19-L27) to the issue. Happy to get other's input as well.

Reply:
Agreed, we should have this probably just be an additional bool argument you can opt into